### DISCUSSION TITLE
Github action stuck at queue

### DISCUSSION THREAD

**Discussion by bjoe88** [community/community#26693] [Actions] [answered]
<p>I have a workflow that stuck at Queue state for a long time. I can’t even cancel the flow. Re-commit also did not help with this problem.  </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1cc14186e3b1456f0a42680bb0092f59f69fe74f.png" data-download-href="/uploads/short-url/46npVqTcHnjEQfOqmd0FTNTwnXp.png?dl=1" title="Screenshot 2019-11-13 at 10.19.57 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/5437454/181099251-997226f0-1523-4427-9ad0-aff6349e3e23.png" alt="Screenshot 2019-11-13 at 10.19.57 AM.png" data-base62-sha1="46npVqTcHnjEQfOqmd0FTNTwnXp" width="690" height="110" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/1cc14186e3b1456f0a42680bb0092f59f69fe74f_2_690x110.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/1cc14186e3b1456f0a42680bb0092f59f69fe74f_2_1035x165.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/1cc14186e3b1456f0a42680bb0092f59f69fe74f_2_1380x220.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1cc14186e3b1456f0a42680bb0092f59f69fe74f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2019-11-13 at 10.19.57 AM.png</span><span class="informations">1548×249 25.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by damageboy** [comment]
<p>Same here.</p>
<p>Builds won’t start, can’t be canceled.</p>

**Comment by tonynv** [comment]
<p>Same here. :-/</p>
<p>I was trying to push for pushing to use it for all our projects.  I know its beta but… Not helping my case</p>

**Comment by kudaba** [comment]
<p>Same here, though I have seen builds working in public repos, mine is private, not sure if that makes a difference.  Failed build (broken yml) is somehow failed and queued for the last 13 hours and the build after it is just pending and not starting.</p>
<p>Edit: Turns out that in my case there was still something broken with my yml file. Fixing that allows new builds to run.</p>
<p>Specifically I had this:</p>
<pre><code>    steps:
    - uses: actions/checkout@v1
      with:
        submodules: recursive
# missing name: here 
      run: .\generate_projects.bat /vs2019
</code></pre>

**Comment by partydrone** [comment]
<p>I saw the same problem. I reached out to GitHub, and it’s a known issue they are working on.</p>

**Comment by xishec** [comment]
<p>same here :cry:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/65ba0b62cdc6406b656bf15484cef4288d7a8b89.png" data-download-href="/uploads/short-url/evUP1ueSj7wUzieKxsFWoRXS3tn.png?dl=1" title="Screen Shot 2019-11-13 at 12.26.00 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/41704009/181099258-4fb5bb9d-4783-4ee5-8339-928a71c1c725.png" alt="Screen Shot 2019-11-13 at 12.26.00 PM.png" data-base62-sha1="evUP1ueSj7wUzieKxsFWoRXS3tn" width="690" height="443" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/65ba0b62cdc6406b656bf15484cef4288d7a8b89_2_690x443.png, https://aws1.discourse-cdn.com/github/optimized/2X/6/65ba0b62cdc6406b656bf15484cef4288d7a8b89_2_1035x664.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/6/65ba0b62cdc6406b656bf15484cef4288d7a8b89_2_1380x886.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/65ba0b62cdc6406b656bf15484cef4288d7a8b89_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-11-13 at 12.26.00 PM.png</span><span class="informations">2652×1706 550 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by bjoe88** [comment]
<p>Now my action has been working perfectly. But the 2 builds that stuck can’t be canceled or deleted. <br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/de91a7ee7ee2c1f58a55810fd2163268de2081e9.png" data-download-href="/uploads/short-url/vKW41DVP1kzwODQNbf8tAJpDnhT.png?dl=1" title="Screenshot 2019-11-14 at 11.42.25 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/5437454/181099261-3c6cf40f-0e1a-4883-8191-2a182dd42344.png" alt="Screenshot 2019-11-14 at 11.42.25 AM.png" data-base62-sha1="vKW41DVP1kzwODQNbf8tAJpDnhT" width="690" height="161" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/de91a7ee7ee2c1f58a55810fd2163268de2081e9_2_690x161.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/de91a7ee7ee2c1f58a55810fd2163268de2081e9_2_1035x241.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/d/de91a7ee7ee2c1f58a55810fd2163268de2081e9_2_1380x322.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/de91a7ee7ee2c1f58a55810fd2163268de2081e9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2019-11-14 at 11.42.25 AM.png</span><span class="informations">1555×364 39.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by rcasperson-jc** [comment]
<p>Do you have a link to track the issue?</p>

**Comment by weide-zhou** [comment]
<p>Hi, </p>
<p>It’s a known issue that github is working on, please contact private support at <a href="https://support.github.com/contact" rel="nofollow noopener">https://support.github.com/contact</a>, thanks.</p>

**Comment by pgmabv99** [comment]
<p>Any news , anyone ?</p>
<p>thanks</p>

**Comment by will-molloy** [comment]
<p>Same here.</p>
<p>Even rolled back to older commit which the build was passing on, and it didn’t even run (stuck in queue!).</p>
<p>ALSO builds in other repositories (where I didn’t even commit, just re-ran them) are stuck in queue.</p>
<p>Must be a GitHub issue.</p>

**Comment by nvengal** [comment]
<p>I’m seeing similar issues. Looks like github actions is seeing degraded performance right now. Hopefully it will clear up as this gets fixed.</p>

**Comment by saska** [comment]
<p>Also encountering this issue, builds ultimately file to some kind of timeout. Copypasting all the error messages I’ve received here so googlers find the thread:</p>
<p>GitHub Actions has encountered an internal error when running your job.</p>
<p>There was a failure in sending the provision message: A timeout occurred while sending request to the remote provider.</p>
<p>There was a failure in sending the provision message: Unexpected response code from remote provider InternalServerError</p>
<p>Provisioning request delayed or failed to send 5 time(s). This is over the limit of 3 time(s).</p>

**Comment by atoomic** [comment]
<p>I’m seeing the exact same issue right now,</p>
<p>hard reset to previously passing commits now fails… </p>
<p>at least <a href="http://status.github.com" rel="nofollow noopener">status.github.com</a> shows actions as degraded</p>

**Comment by ryanzhang-oss** [comment]
<p>Same here, all builds stuck and CI failed with </p>
<pre><code>GitHub Actions has encountered an internal error when running your job.
</code></pre>

**Comment by brookmg** [comment]
<p>My builds are also stuck in the queue. But fails after 15mins or so saying </p>
<pre><code>GitHub Actions has encountered an internal error when running your job.
</code></pre>

**Comment by BernalCarlos** [comment]
<p>any issue link? has it been resolved? What’s time can be considered normal? Mine has been stuck for 10 minutes for the moment, which is already weird to me.</p>

**Comment by empire-jlee** [comment]
<p>Is this issue still occuring for everyone?</p>

**Comment by stevelizcano** [comment]
<p>I’ve been getting it the last fifteen minutes</p>

**Comment by empire-jlee** [comment]
<p>Glad to know I’m not the only one. This issue must be coming from Github’s end</p>

**Comment by saminwankwo** [comment]
It's 2022 and I am still having this problem, actions has been running for over 2hours

**Comment by MarkosMuche** [comment]
Same here, queued for 8 mins so far.

**Comment by wilbertcaba** [comment]
It's a known issue by Github. Check: https://www.githubstatus.com/

Hopefully they resolve soon. Subscribe to this issue and I assume they will notify us when resolved:

![Screen Shot 2022-08-24 at 2 19 21 PM](https://user-images.githubusercontent.com/36094085/186493948-92d038e4-913d-43b6-81b3-b02952617f56.png)

**Reply by varunchillara27** [reply]
Thanks for this, looks like it is working now

**Comment by Adam-it** [comment]
nope. does not seem to be resolved. I still got one flow in progress which I cannot cancel and is stuck in Queued

**Reply by Adam-it** [reply]
ok I opened a ticket as for it as it's definitely still not working in our case 😔

**Comment by patbi** [comment]
Hello, Hello.

I'm building [this project](http://thenavigo.com/) using github-actions, since this morning I have this problem.

github runner not picking up job.

any suggestions?

**Comment by hwweber** [comment]
i'm also having this problem intermittently as of 3 days ago.

**Comment by DarioMarzella** [comment]
I am having the same issue [here](https://github.com/X-lab-3D/PANDORA/actions/runs/2998840136), but quite interestingly it happens to me only for actions that include two separate jobs. The first job runs like a charm as always, the second one stays in the queue forever.

**Reply by PikachuEXE** [reply]
I also got jobs stuck in queue since about 5 hours ago
https://github.com/PikachuEXE/contracted_value/actions
![image](https://user-images.githubusercontent.com/1018543/188647861-3a544cae-eaa2-435f-a867-f267f10980aa.png)

**Comment by cdaf** [comment]
If anyone else stumbles across, i.e. a working action pipeline stops working, check for a deprecated agent definition, e.g. Windows Server 2016 no longer supported,

```
    runs-on: windows-2016
```

**Reply by roomrys** [reply]
Likewise for
```
runs-on: ubuntu-18.04
```
per this [article](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/).

Updated the ubuntu image and things seem to be working as usual again.

**Comment by oxbo-andre** [comment]
Have it today also, when I do a manual trigger what worked yesterday and today nothing is happening.

**Reply by alexswearingen** [reply]
I also am running into this issue today, not sure if this is a coincidence on a two-year old thread...

**Reply by matthewhutchings** [reply]
also having issues

**Reply by kluevandrew** [reply]
same for me

**Reply by chrismulholland** [reply]
same.

**Reply by rwoodard-prog** [reply]
Probably due to partial outage in GitHub Actions:

https://www.githubstatus.com/

```
Update - We are seeing recovery in Actions and will update status when all runs are no longer delayed.
Oct 05, 2022 - 17:03 UTC

Update - We continue to see recovery and have added capacity to speed recovery time. We still expect to see slower than normal Actions job runs and webhook deliveries.
Oct 05, 2022 - 16:12 UTC

Update - Actions is experiencing degraded performance. We are continuing to investigate.
Oct 05, 2022 - 15:04 UTC
```

**Reply by sluxzer** [reply]
GitHub Service Status
Actions is currently status yellow; with an update as of about 2 hours ago. This may affect Actions behavior and performance.
Please check [GitHub Status](https://stspg.io/jlh3qwzg6q7s) for more information.


additional information for you whos not aware yet.

**Comment by nhtruong** [comment]
We're seeing the same issue for the past 2 days. No new workflows can be queued and the stuck workflows can't be canceled. Github Status has been green across the board.

**Reply by adrime92** [reply]
Same here since friday, today it's still happening.

**Comment by rostomy** [comment]
same with me today i've been trying so slove the issue for the past 10 hours, but no results, can anyone help me

**Comment by avantgardnerio** [comment]
+1 and as usual https://www.githubstatus.com/ shows all green (for now at least...)

**Comment by arthurnap24** [comment]
Hi all, experienced this yesterday on my self-hosted runners and got it solved. My problem was:
- The same as OP
- Ran the actions runner network connectivity script and is passing all
- Installed/re-installed Github actions runner service to no avail

Solution:
- Removed self-hosted runners, and re-added them
- Re-started the service
- Check un-check the tags of my self-hosted runners under `Settings`->`Actions`->`Runners`->`RUNNER-NAME`

The workflow finally got picked up after resetting the tags.

Things to consider:
- Not so sure if you need to remove the runners or just mess around with the tags. So mess with the tags first before removing to avoid extra work.

Thanks,
AN

**Reply by mehmetcankamar** [reply]
same problem here., your solution works. thanks a lot.

**Reply by arthurnap24** [reply]
Awesome! Thanks for upvote.

**Comment by erikvanzijst** [comment]
Same problem here. Not using self-hosted runners: #1856101

**Comment by jaroslavtyc** [comment]
First time manually rerun an action (PHP composer + FTP deploy).

Stuck on _Queued_ state (changed secrets used in the action - maybe is it linked?).

**Comment by snej** [comment]
Actions in a private repo have stopped running in the past few days — they’re all stuck at Queued.

I thought I might have used up my quota of free minutes (though I didn’t get a notification) so I set up a $20/mo spending limit, but that didn’t help.

FWIW, these actions use both Mac and Linux runners, for C++ builds.

**Reply by arthurnap24** [reply]
Hi, have you tried my solution above? https://github.com/orgs/community/discussions/26693#discussioncomment-3979889

**Reply by snej** [reply]
I don’t use self-hosted runners.

**Comment by xiaohk** [comment]
Having the same issue now :( It has been in queue for 10+ minutes.

**Reply by DiegoLimaDev** [reply]
I'm facing this issue as well. I believe started something like 15 minutes ago

**Reply by AnatolyBuga** [reply]
> Having the same issue now :( It has been in queue for 10+ minutes.

Same

**Reply by rubanero14** [reply]
Same here. 10mins+ and still counting.

**Reply by chrismin13** [reply]
mine just switched to deployement_in_progress!

**Comment by Ainuraut** [comment]
I am having the same issue with the deployment!!!

**Comment by mayar-ariss** [comment]
Same issue here, no self-hosted runners. It outputs a repeated message:
Current status: deployment_queued

**Comment by RELNO** [comment]
looks like a GH actions issue: https://www.githubstatus.com/

**Reply by Niek** [reply]
Specific issue: https://www.githubstatus.com/incidents/n18j5g7wpg0s

**Comment by FaizanMS** [comment]
![Screenshot 2023-01-19 at 22-24-36 pages build and deployment · faizan-smit_saylani-welfare-clone@778d36e](https://user-images.githubusercontent.com/79311662/213516288-3c7406eb-3c06-4917-b4cd-fa8dd486c6d0.png)

Stuck here while deploying a static web page, any recommendations/solutions?

**Reply by RELNO** [reply]
I'm seeing the same. Get a coffee and let @github-staff do the magic ;)

**Comment by mustsee** [comment]
Same here :)

**Comment by silverqx** [comment]
Status here https://www.githubstatus.com/

**Reply by marcelo-earth** [reply]
Woow. Thank you!

**Reply by alinematilux** [reply]
Thanks for sharing this!

**Comment by Bosh-Kuo** [comment]
Same here @@

**Comment by alessiolapenna** [comment]
Same here...i was restarting my runner while it was being queued and now it's stuck on queued. Even new actions just get in the waiting state because the queued action is taking the spot. Restarting the runner didn't help.

**Comment by vbest-shimizu** [comment]
Same here.
Job remains stuck in queue in GitHubActions for several hours Job executed in GithubActions remains stuck in queue and does not execute after 3 hours.
When I run Cancel, I get "Workflow cancel request succeeded." and several hours have passed without any change.

**Comment by kennethlynne** [comment]
We're currently experiencing this now. Stuck in queued for 1hr+ now. We're not using self-hosted runners or anything special

**Reply by IceflowRE** [reply]
I top you by an action starting since 21months.

**Comment by glacials** [comment]
To everyone arriving here from Google: make sure your `runs-on` is valid (e.g. `ubuntu-latest`, not `ubuntu`), as the job will get stuck in the queue if it is not.

**Comment by jmyers-code** [comment]
It's stuck on Queue. Help a brother out

**Reply by antoniolmcandido** [reply]
This is a general problem

**Comment by antoniolmcandido** [comment]
03/02/2023 21:20hs - the problem happened again

**Reply by mikesocorecc** [reply]
Estoy en las mismas, se quedó atascado en Queued

**Comment by ShinguYui** [comment]
Same here, How should I solve this?
I can't even cancel this.
The error message 'Failed to cancel workflow.' appears.

**Reply by BachWV** [reply]
Same. It seems like an incident with github actions. https://www.githubstatus.com/

**Comment by MarkParker5** [comment]
Same problem. It worked two hours ago, but now it doesn't. Looks like an internal github issue.

**Comment by antoniolmcandido** [comment]
https://githubstatus.com

**Comment by chojs23** [comment]
Seems like github action has some issues. It works now.

**Comment by youngjun0627** [comment]
It still doesn't seem to work well on a self-hosted runner.
<img width="1287" alt="image" src="https://user-images.githubusercontent.com/57058726/222634585-f869963f-7e2e-45b0-a343-1e8ae316261b.png">

**Comment by ejain** [comment]
Still have a workflow run that's stuck in _Queued_ state (and that can't be cancelled), even though the outage was marked as resolved several hours ago... Opened a support ticket.

**Reply by ejain** [reply]
They had to manually clear out the stuck run, everything back to normal.

**Comment by ettaboyle** [comment, marked as answer]
Hi all 👋🏻 **Current Status:** [Investigating reports of degraded performance](https://www.githubstatus.com/incidents/sr7zptjd0b07)

Thanks for sharing and we appreciate your patience. You can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for upd

**Reply by Vadorequest** [reply]
@ettaboyle I've been stuck for 3 days, none of our actions are being started.
I've opened a ticket, but this is concerning, and the Status Page says everything is alright :/

![image](https://user-images.githubusercontent.com/3807458/230760365-e88b771d-438f-40b6-a3dc-8bd93cd78989.png)

**Reply by retendo** [reply]
Happens for our project as well at the moment

**Comment by khrulenko** [comment]
Hi, I have the same issue with stack queued deployments, 
could you help me, please?

**Reply by dimka9910** [reply]
looks like currenly there is action degradation https://www.githubstatus.com/

**Reply by cherrygot-personal** [reply]
Same here. Actions are queued forever. Earlier I was struggling with Github action not being able to fetch the repo due to the change in their public RSA key. And now this. Looked like I broke something but I guess it's global outage.

**Reply by ghost** [reply]
It seems like they're working to fix it

**Reply by antoniobarolo** [reply]
I had the same concern because I started working on my project today right at the same time the reports began. How long do these kinds of problems tend to last?

**Reply by mstmdev** [reply]
Same here...

**Reply by Tekkaee** [reply]
The same

**Reply by gescobedo** [reply]
same please help

**Reply by raphlee** [reply]
Here too, It's been about an hour.

**Reply by rujutashinde** [reply]
same here , still the actions are seen as queued

**Reply by anykeyist** [reply]
> Hi, I have the same issue with stack queued deployments, could you help me, please?

https://github.com/orgs/community/discussions/26693#discussioncomment-5184759

**Reply by KaroIex** [reply]
It is a general problem now, we must wait https://www.githubstatus.com/

**Reply by Saaharjit007** [reply]
It seems like the status has been restored but my job is still stuck in queued state.

**Reply by gescobedo** [reply]
Try adding a small change  and deploy

**Reply by samernasif** [reply]
same issue here

**Reply by denisyilmaz** [reply]
same issue here

**Reply by GiveMeAnApple99** [reply]
Same here, it has stucked for 20 hours...

**Reply by Vadorequest** [reply]
I've been stuck for a few days, tried to stop/start but didn't solve it yet.

![image](https://user-images.githubusercontent.com/3807458/230727153-c2b17bfa-b809-4024-934a-4461e1c0f307.png)

**Reply by azinit** [reply]
same problem!
https://github.com/feature-sliced/documentation/actions

**Reply by azinit** [reply]
> Try adding a small change and deploy

Not working for me :(

**Comment by Akash1134** [comment]
Hi everyone, 

Sorry for the inconvenience caused due to the stuck workflows. For those who are still facing issues with Action stuck in the queue, let me know so that I can work with the team internally and get this sorted.

**Reply by azinit** [reply]
Hi, @Akash1134 
I still have problems with stucked actions 😢 
https://github.com/feature-sliced/documentation/actions/runs/4654870166

**Reply by Vadorequest** [reply]
@Akash1134 Ours are stuck too, since 3 days+

https://github.com/UnlyEd/TFP-v4/actions/runs/4656625818

Thank you!

**Reply by taoning** [reply]
Hi @Akash1134 Ours has been stuck for a week now. https://github.com/LBNL-ETA/Radiance/actions/runs/4653134790
Thank you!

**Reply by Vadorequest** [reply]
I got a reply from the support. My issue seems to be related to https://github.com/actions/runner-images/issues/6002

I think it might be similar for other folks here, the fact that there are no error message makes believe it is a bug, but it might be "on purpose", although the lack of proper error message sucks.

![image](https://user-images.githubusercontent.com/3807458/231072948-97837159-231b-47ce-97b7-2b3352d90b12.png)

**Reply by alsace-research** [reply]
I am experiencing the same problem (queued Github actions + GitHub pages) for the following repository: https://github.com/alsace-research/alsace-research.github.io

Github Actions: https://github.com/alsace-research/alsace-research.github.io/actions

Love hosting on Github pages -- but this issue will have me looking at alternatives soon.  

-- Update since last post:  
 - I made a fresh commit & deployed successfully
 - Issue appears to be resolved... thanks!

**Reply by Dyljyn** [reply]
Same here since a couple of days: https://github.com/Nalinija/autodata-angular/actions/runs/4680399729

**Reply by azinit** [reply]
> ![image](https://user-images.githubusercontent.com/3807458/231072948-97837159-231b-47ce-97b7-2b3352d90b12.png)

@Vadorequest Thank you! The changes below helped me:

```diff
- runs-on: ubuntu-18.04
+ runs-on: ubuntu-latest
```

https://github.com/feature-sliced/documentation/commit/f50d009bad083081f093f46926a5e077cb4f9a01

**Comment by estevaolucas** [comment]
I'm getting the same issue with `ubuntu-latest`

```
Requested labels: ubuntu-latest
Job defined at: squareup/dashboard-fulfillment/.github/workflows/semantic-pull-request.yml@refs/heads/main
Waiting for a runner to pick up this job...
```

Started one hour ago. Status page doesn't show any degradation

**Comment by agil3st** [comment]
I'm getting the same issue with `self-hosted`
![image](https://user-images.githubusercontent.com/29949806/236782179-52bc7af6-49c3-47b2-b837-069aab4aec26.png)

**Reply by dongsupark** [reply]
Same issue here, stuck since ~24 hours.
```
Requested labels: self-hosted, debian, build, x64
Job defined at: dongsupark/flatcar-scripts/.github/workflows/update-sdk.yaml@refs/heads/main
Waiting for a runner to pick up this job...
```
I am not even able to cancel that. Tried both UI and CLI, nothing works.

**Reply by Rickcau** [reply]
Same for me, sitting in queued status for 3hrs

**Reply by srg1177** [reply]
This is still actual, job is queued, there is not output in console but the job is actually running under the hood as my web-hook triggered somehow...

**Reply by Kehindecodes** [reply]
still have the same issue over 24 hours now

**Reply by Charizard13** [reply]
any ideas or solutions?

**Comment by 95-martin-orion** [comment]
The symptoms described in this issue can occur if your runner image is no longer supported. This happened to my project, which has been using the recently-deprecated `macos-10` image. If you are having this issue, verify that the runner image your action is using is one of the [supported versions](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) (e.g. `ubuntu-latest` or `macos-11`). If the runner image you use does not appear, try updating to one of the versions listed.


Hope this helps!

**Comment by sahana-1** [comment]
We are using GitHub enterprise and facing the same issue. I have used runners: windows-latest and ubuntu-latest. Neither of them worked. Build is stuck for more than 2 days in the Queued state.

**Comment by GavinMilbank** [comment]
I had copied a static.yml file with this chunk of code in it, which was blocking concurrent jobs.  Removing it unblocked things.
```{yaml}
# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
#concurrency:
#  group: "pages"
#  cancel-in-progress: false
```

**Comment by eivindml** [comment]
Actions how now been stuck for 40 minutes. 

<img width="1357" alt="Skjermbilde 2023-06-26 kl  16 20 21" src="https://github.com/community/community/assets/202450/37c3cb87-4505-4912-8818-8eaec18f479a">

**Reply by benjamin-mogensen** [reply]
Ours are also stuck now

**Reply by Akash1134** [reply]
Hi @eivindml and @bjoe88 ,

I have highlighted this to our engineering team. Thanks for reporting!

**Comment by Jinwook94** [comment]
A workflow build has been in progress for over 40 hours.

<br>

<img src="https://github.com/community/community/assets/44575214/85647be5-41fc-4b68-82a2-fc9c09f3dbb1" width="550">


https://github.com/Jinwook94/bootme/actions/runs/5368195567
https://github.com/Jinwook94/bootme/actions/runs/5368352622

<br>

The workflow has been deployed without any issues, with all steps working correctly, but the Jobs tab on the workflow page is still showing a 'build' status, while the favicon on that tab is showing a check mark for completion.

<br>

![image](https://github.com/community/community/assets/44575214/63326b51-6cf8-4e20-b9b2-68a4c5042364)

<br>

The all workflows page shows that the workflow action is complete.

![image](https://github.com/community/community/assets/44575214/f964bef8-6b26-470f-b84d-d1fbae94dc79)
https://github.com/Jinwook94/bootme/commit/c640f20eb792082fa546939f34eef675070531bc

<br>

Still showing 'build' status on the commit page too.

https://github.com/Jinwook94/bootme/tree/4d130350601701d54e60b308ef6f4b3879d05507/.github/workflows
This is the workflow file at that point in time, and I didn't make any modifications in those commits.

<br>


GitHub Actions are great. Hope this issue fixed.

**Comment by hamilton-lima** [comment]
So far stuck for 2 hours...
![image](https://github.com/community/community/assets/2771784/e9e15e5d-08d6-4767-a0b9-5f8330909152)

**Reply by hamilton-lima** [reply]
10 hours now...
![image](https://github.com/community/community/assets/2771784/164fe17a-78ba-4b41-b0dd-bfd98f37bc0e)

**Reply by Akash1134** [reply]
Hi @hamilton-lima ,

Sorry that Actions is not running as expected. It seems like your workflow might be stuck in the background preventing the run.

The best place is to open a ticket on our [Support page](https://support.github.com/contact) and get this sorted. Please open a ticket with them :)

**Reply by hamilton-lima** [reply]
I was able to fix the issue by changing the version of the `run-on` machine from `ubuntu-18.04` to `ubuntu-latest`. 
![image](https://github.com/community/community/assets/2771784/df2fde2f-9eea-4229-a558-a103958b2dbb)

**Comment by farih96** [comment]
i'm using a self runner and i encountered the same error, any ideas how it can be fixed ?

**Reply by youngjun0627** [reply]
We had an issue where an unknown network issue between GitHub and the runner caused the latest version of the runner to take a long time to download, causing the workflow to stay in the queue. We solved this by enabling the `disableupdate` option and creating an automated workflow to update the runner version. See [here](https://github.blog/changelog/2022-02-01-github-actions-self-hosted-runners-can-now-disable-automatic-updates/).

**Comment by seamaner** [comment]
stuck for 2weeks:

https://github.com/seamaner/seamaner.github.io/actions/runs/6079696146

**Comment by ccoelho-dzk** [comment]
Currently stuck waiting for a macos runner for over an hour

**Comment by Miniontoby** [comment]
Currently waiting for over 17 hours over at https://github.com/Miniontoby/plaintext-song-parser.js/actions

Please github staff, if you read this, remove the three queued actions. I cannot cancel them

**Comment by AnderUstarroz** [comment]
Same issue, I have a task queued forever which cannot be canceled. This is definitely a bug on Github side, I don't see a valid reason for keeping tasks in a limbo like this.

**Comment by benjamin-mogensen** [comment]
We are having lots of PR GitHub Actions getting stuck now. Anyone else having same problems?

**Comment by Sorbh** [comment]
same issue, deploy stuck in queue for last 15 hr. Can't be cancel either.

**Comment by AnderUstarroz** [comment]
For future reference, if you are unable to cancel a workflow run from the `Actions UI` or cancel REST API endpoint, you can force cancel the workflow run through the REST API.
 
For convenience, here is a curl command you can use to force cancel a workflow run:
 

```
curl -IvL \
  -X POST \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/<OWNER>/<REPO>/actions/runs/<RUN_ID>/force-cancel
```

**Reply by benjamin-mogensen** [reply]
That is nice! Thanks for sharing @AnderUstarroz

**Comment by benjamin-mogensen** [comment]
Stuck again :( can GitHub soon fix the root cause here?

**Reply by Lctech-Jerry** [reply]
Yesterday, it also got stuck for a while, but it went back to normal soon after. Now it’s starting to get stuck again.

**Comment by TengriKut1923** [comment]
I'm having the same problem. It was because of yesterday's outage. In the past, this problem was resolved within 24 hours.
https://github.com/orgs/community/discussions/49074

**Comment by BhargavaTR** [comment]
Unable to cancel the workflow queue

**Comment by Justina-03** [comment]
Same here......I have a GitHub Actions workflow that is stuck and won't finish running, even after several attempts to cancel it. Even though I’ve tried to cancel multiple times, it still shows as running, and the 'failed to cancel' message appears. And when i push the new commits, they are working and building successfully, but this one specific workflow run is stuck and won’t finish.

**Comment by TheRealSmeddy** [comment]
Seems like the issue is still up, never heard anything about "stuck actions", faced them today, and yes It can't be canceled, status pages shows everything is up and running

**Comment by vivaanheycoach** [comment]
FACING THE SAME ANY UPDATE ON THIS ??

**Comment by ofer-ws** [comment]
Github runners are stuck, everything is queued, there's nothing in the status page.

**Comment by MahdadGhasemian** [comment]
All runners ware queued now!

**Comment by kashifkhan0771** [comment]
Time for a break :)

**Comment by TheRealSmeddy** [comment]
Guys, try to retrigger the actions, and rely on the data shown in the job section (In my case Build & Deploy).
![image](https://github.com/user-attachments/assets/0c3a4916-2c84-4150-8727-44a25f6d037a)

The indicator and status may be shown as "Pending/Queue" but the job could be run, click on it and check if it has executed the action.
In my case after several runs, it was still showing pending but the action was executed

**Comment by Rafisto** [comment]
The issue seems to be back. Time for a coffee :)

**Comment by thinguyen0** [comment]
me too!

**Comment by BilsonTan** [comment]
Same issue for me as well!

**Comment by Kaviilee** [comment]
same

**Comment by btcs-Son** [comment]
same here

**Comment by JarrydSaf** [comment]
Same here

**Comment by Alvin2281** [comment]
Same here. can't cancel . also stuck in queue

**Comment by vijayideya** [comment]
same here github actions are being struck is any other facing the same

**Comment by mdimitrov** [comment]
Actions UI is totally broken. Jobs steps seems to be still executed without problems.

**Comment by Rafisto** [comment]
I have just received an email of a cancelled action which I requested through the UI about 30 minutes ago. The runners work, but seem to be under heavy load.

**Comment by TD-jameszhang** [comment]
same here github actions are being struck is any other facing the same

**Comment by jfunction** [comment]
They are aware of the issue https://www.githubstatus.com/

**Comment by suraj-kd-pro** [comment]
My boss is saying to fix GitHub right now otherwise you will be fired :(

**Reply by liam-baker-tpc** [reply]
Good luck 🙏

**Reply by ParthShindenovus** [reply]
happy diwali bro enjoy

**Reply by manjunathp-jm** [reply]
you can say FO

**Reply by swgu98** [reply]
Help！me, too

**Comment by ParthShindenovus** [comment]
Happy Diwali

**Comment by adarshnovuscode** [comment]
Happy Halloween

**Comment by danialmecja** [comment]
Sorry my bad folks, pushed something to prod about an hour ago via actions...

**Comment by vinal-novuscode** [comment]
Happy Kali Chaudas and Halloween

![Happy-Kali-Chaudas-Wishes-Greeting-Images-Photos-Picture-1-2002286146](https://github.com/user-attachments/assets/5f401eed-bc14-4585-abd6-b0b71dba96c4)

**Comment by boymep** [comment]
Someone please ping us if see the problem resolved 🤖

**Comment by btcs-Son** [comment]
it still runs the workflow, but the UI did not update the status. I checked the job and it ran

**Comment by anilrajrimal1** [comment]
yup, now working !!
just 56mins :turtle:

**Comment by admin-kinetic** [comment]
Cleaning up orphan processes .... taking forever :(

**Comment by mcitem** [comment]
hello,
![屏幕截图 2024-10-30 164206](https://github.com/user-attachments/assets/9059d2a3-b0c6-47ed-bf3e-f1c04ad19515)

**Comment by boymep** [comment]
UPD: It has a long time to pass the pipeline but it seems to be working for now

**Comment by nghiapq** [comment]
It's still stuck in 2025 with more than 10 jobs that cannot be canceled or built.
